apps/beeai-server/src/beeai_server/domain/constants.py

```diff
@@ -15,7 +15,7 @@ class _Undefined(Enum):
 undefined = _Undefined.undefined
 Undefined: TypeAlias = Literal[_Undefined.undefined]  # noqa: UP040
 
-DEFAULT_AUTO_STOP_TIMEOUT: Final[timedelta] = timedelta(minutes=5)
+DEFAULT_AUTO_STOP_TIMEOUT: Final[timedelta] = timedelta(minutes=20)
```

Comment:
Please update `CreateProviderRequest` and `PatchProviderRequest` to include this number in the allowed range (currently maximum is 10 minutes)